### ISSUE TITLE
[V2.0.6] Paging bug: +10000 products

### ISSUE THREAD

**Issue by leguigou** [akeneo/pim-community-dev#7184]
**I'm reporting a Bug**
[akeneo-pim-system-info_2017-11-20_16_15.txt](https://github.com/akeneo/pim-community-dev/files/1488099/akeneo-pim-system-info_2017-11-20_16_15.txt)
![capture](https://user-images.githubusercontent.com/32835592/33025414-3792a91c-ce0e-11e7-838f-97c4572909f3.PNG)

Hello,

Beyond 10000 products, we have an error when we click on the pagination of the last pages.
Beyond 400 pages we get the following error :
Server error
Error! Incorrect server response.

I tested on your demo server, same problem !

cordially

**Comment by LaureBro** [comment]
Hello, 
This is a problem linked to Elastic Search, we'll improve the pagination to have this issue fixed.
We'll keep you updated.
Kind regards, 
Laure

**Comment by lars85** [comment]
@leguigou, for the moment (in the meanwhile) you can increase the limit in the elastic search index:
`curl -XPUT 'http://localhost:9200/_all/_settings?preserve_existing=true' -d '{
  "index.max_result_window" : "150000"
}'`

**Comment by AngelVazquezArroyo** [comment]
@leguigou, a possible solution is adding a new .yml for example elastic_index_configuration.yml under app/config with the followig content:
`settings:
    max_result_window: 500000`
and then in pim_parameters.yml you must only insert a new line with your new .yml config file target, like that
`# Files used as configuration for the Elasticsearch index
    elasticsearch_index_configuration_files:
        - '%kernel.root_dir%/../vendor/akeneo/pim-community-dev/src/Pim/Bundle/CatalogBundle/Resources/elasticsearch/index_configuration.yml'
        - '%kernel.root_dir%/../app/config/elastic_index_configuration.yml'`

**Comment by AngelVazquezArroyo** [comment]
@leguigou, a possible solution is adding a new .yml for example elastic_index_configuration.yml under app/config with the followig content:
```
settings: 
      max_result_window: 500000
```
and then in pim_parameters.yml you must only insert a new line with your new .yml config file target, like that
```
# Files used as configuration for the Elasticsearch index
    elasticsearch_index_configuration_files:
        - '%kernel.root_dir%/../vendor/akeneo/pim-community-dev/src/Pim/Bundle/CatalogBundle/Resources/elasticsearch/index_configuration.yml'
        - '%kernel.root_dir%/../app/config/elastic_index_configuration.yml'
```

**Comment by Doodoune** [comment]
Hi! 

This issue was solved in the patch 2.0.22 of the PIM.
Please upgrade your PIM.
I close this issue, feel free to reopen it if needed.

Regards,
